.github/workflows/start-vulnerability-updater.yaml

```diff
@@ -0,0 +1,58 @@
+name: Vulnerability Updater
+on:
+  push:
+    branches:
+      - "yli3/**"
+  schedule:
+    - cron: "0 */3 * * *"
+
+jobs:
+  read-versions:
+    runs-on: ubuntu-latest
+    outputs:
+      matrix: ${{ steps.output-matrix.outputs.matrix }}
+    steps:
+      - name: Checkout code
+        uses: actions/checkout@v3
+
+      - name: Generate matrix JSON
+        id: output-matrix
+        run: |
+          EOF=$(dd if=/dev/urandom bs=15 count=1 status=none | base64)
+          echo "matrix<<$EOF" >> $GITHUB_OUTPUT
+          sed -n '/^\s*\(#.*\)\?$/!p' scanner/updater/version/RELEASE_VERSION | jq -Rs '{ versions: split("\n") }' | tee -a "$GITHUB_OUTPUT"
+          echo $EOF >> $GITHUB_OUTPUT
+
+  output-versions:
+    needs: read-versions
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        version: ${{fromJson(needs.read-versions.outputs.matrix).versions}}
+    env:
+      ROX_PRODUCT_VERSION: ${{ matrix.version }}
+    steps:
+      - name: Authenticate with Google Cloud
+        uses: google-github-actions/auth@v1
+        with:
+          credentials_json: ${{ secrets.GOOGLE_SA_CIRCLECI_SCANNER }}
+
+      - name: Set up Cloud SDK
+        uses: google-github-actions/setup-gcloud@v1
+
+      - name: Download and unzip file
+        run: |
+          DOWNLOAD_URL="https://github.com/stackrox/stackrox/archive/refs/tags/${{ env.ROX_PRODUCT_VERSION }}.zip"
+          FILE_NAME=$(basename "$DOWNLOAD_URL")
+          wget "$DOWNLOAD_URL" -O "$FILE_NAME"
+          unzip "$FILE_NAME" -d unzipped-dir
+          cd unzipped-dir/stackrox-*
+          if [ ! -d "scanner" ]; then
+            echo "Scanner directory not found. Terminating current matrix step."
+            exit 0
+          fi
+          
+          cd scanner
+          go run cmd/updater/update.go -outputDir=${{ env.ROX_PRODUCT_VERSION }}
+          gsutil cp -r "${{ env.ROX_PRODUCT_VERSION }}" "gs://scanner-v4-test/"
+        
```

Comment:
@tommartensen I wonder what's the best format to add the failure check? 
```
- name: Send Slack notification on workflow failure
      if: failure()
      run: |
        curl -X POST -H 'Content-type: application/json' --data '{"text":"<!subteam^S04S96AAVND|acs-scanner-primary> Failure in Scanner V4 Updater Workflow: Download and Upload Repository Mappings"}' ${{ secrets.SLACK_ONCALL_SCANNER_WEBHOOK }}
```

Reply:
that looks ok. all subsequent steps without the failure() condition will be skipped.